Gateway/Request/MotoMerchantAccountDataBuilder.php

```diff
@@ -50,6 +50,7 @@ public function build(array $buildSubject)
         }
 
         $request['body'] = $this->adyenRequestsHelper->buildMotoMerchantAccountData($motoMerchantAccount);
+        $request['clientConfig']['storeId'] = $payment->getMethodInstance()->getStore();
```

Comment:
Does this mean that the Magento store ID will be sent in the request?

Reply:
Hey @acampos1916, thanks for the review! No, that doesn't mean it is a part of the request. Actual request is being built around `$request['body']`. However, `$request['clientConfig']` this array is for helper parameters and we are not sending it in the request.